packages/arb-token-bridge-ui/src/components/common/HeaderMenu.tsx

```diff
@@ -86,7 +86,7 @@ export function HeaderMenuMobile(
       {({ open }) => (
         <div className="w-full">
           <Disclosure.Button
-            className={`arb-hover flex w-full items-center justify-start p-4 ${
+            className={`arb-hover flex w-full items-center justify-start px-6 py-3 ${
```

Comment:
Please review the padding changes to make it consistent with the padding of all dropdowns (like network selector) in the accordion.

Reply:
updated padding and icon size across the header according to figma